### ISSUE TITLE
Backend code

### ISSUE THREAD

**Issue by karim23657** [jsmsj/megatree#3]
Hello, Thanks for your great project @jsmsj and @Flothaboss1000 .
Could you please share backend code? (for this https://megajsmsj-1-n9032891.deta.app/get_mega_stuff)

**Comment by apeoplerson** [comment]
Yes I was just looking at this too. I agree is it possible to get the backend code? Ill pay.

@jsmsj and @Flothaboss1000
